pkgs/desktops/gnome-3/extensions/gtile/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, glib }:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-extension-gtile";
+  version = "34";
+
+  src = fetchFromGitHub {
+    owner = "gTile";
+    repo = "gTile";
+    rev = "V${version}";
+    sha256 = "1yka4acmrykh4vdd89yvn6yv89c6q16r72lgi3j48my3jghwjiwp";
+  };
+
+  uuid = "gTile@vibou";
+
+  nativeBuildInputs = [ glib ];
+
+  installPhase = ''
+    mkdir -p $out/share/gnome-shell/extensions/$uuid
+    cp -r * $out/share/gnome-shell/extensions/$uuid
+
+    schemadir=${glib.makeSchemaPath "$out" "${pname}-${version}"}
+    mkdir -p $schemadir
+    cp -r $out/share/gnome-shell/extensions/$uuid/schemas/* $schemadir
+  '';
+
+  meta = with stdenv.lib; {
```

Comment:
Would you be willing to add yourself as a maintainer of the package?

Reply:
I will try to add, don't know exactly the procedure now.

Reply:
Here is a fairly recent example: https://github.com/NixOS/nixpkgs/commit/32341633786403a48138df03f6b8ebc2938366c0

Ideally this PR would then consist of two commits:

- `maintainers: add firmero`
- `gnomeExtensions.gtile: init at 34`

where the second commit can reference the newly added maintainers' entry in the meta attributes like so:

```
meta = with stdenv.lib; {
  ... # other attributes
  maintainers = [ maintainers.firmero ];
};
```

Reply:
Done, thanks for reference.